swap the questions and add code langs (#1921)
enkidevs/curriculum: javascript/core/errors/debugging.md

It can be useful to find out the values of variables when debugging code. 

Some developers will use an alert statement to do this:

```js
alert(myVar);
```

A better way is to use the console.log statement to write the variables value out to the console:

```js
console.log(myVar); 
```

---
## Practice

How can you log variables into the console?

> ```

Reviewer: should we add `js` tags to this code block?
``````suggestion
```js
``````

* `debugger;`
* `addVar`

---
## Revision

> What statement will make the debugger stop at its line?

Author: ```suggestion
What statement will make the debugger when it sees it?
```
